README.md

```diff
@@ -40,21 +41,21 @@ Python client library to quickly get started with the various [Watson APIs][wdc]
 To install, use `pip` or `easy_install`:
```

Comment:
Should we have a note here that versions prior to v3.0.0 can be found in `watson-developer-cloud`?